### PR TITLE
projects: add "Live Protocol Overhead Dissector/Statistics for Gluon"

### PR DESCRIPTION
**Pull request by T-X**
### Checklist for new or updates project ideas

- [x] I added at least one contact option so interested people can get in touch with me.
- [x] Data for my project is valid

### CODE REVIEW

**Comment by T-X** [comment]
(I don't quite understand what the CI's yaml validator want's me to change. At least this site says it's valid? https://yamlchecker.com/)

**Comment by andibraeu** [comment]
unfortunately other_chat_options is just defined as an array of strings

File: _projects/gluon-protocol-overhead-dissector.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+a server. This has already (mostly) been implemented via:
+
+* gluon-statistics-mcast package:
+  * https://github.com/freifunk-gluon/gluon/pull/2367
+* bpfcountd:
+  * https://github.com/lemoer/bpfcountd/
+  * this is used by gluon-statistics-mcast for getting the
+  * this uses libpcap + a (not yet upstream) batman-adv dissector patch
+
+---
```
Comment (andibraeu) [inline _projects/gluon-protocol-overhead-dissector.md:61]: I'm not sure if that works, normally that defines a new document afterwards

**Comment by T-X** [comment]
@andibraeu Ah, gotcha, thanks!

**Pull request merged**